src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/HwFDashboardNotificationsHandler.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.dashboardnotifications;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.callback.DashboardCallbackHandler;
+import uk.gov.hmcts.reform.civil.client.DashboardApiClient;
+import uk.gov.hmcts.reform.civil.enums.FeeType;
+import uk.gov.hmcts.reform.civil.helpers.CaseDetailsConverter;
+import uk.gov.hmcts.reform.civil.launchdarkly.FeatureToggleService;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.service.CoreCaseDataService;
+import uk.gov.hmcts.reform.civil.service.DashboardNotificationsParamsMapper;
+
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.NO_REMISSION_HWF_GA;
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.APPLICANT_LIP_HWF_DASHBOARD_NOTIFICATION;
+import static uk.gov.hmcts.reform.civil.handler.callback.camunda.dashboardnotifications.DashboardScenarios.SCENARIO_AAA6_GENERAL_APPS_HWF_REJECTED_APPLICANT;
+
+@Service
+public class HwFDashboardNotificationsHandler extends DashboardCallbackHandler {
+
+    private static final List<CaseEvent> EVENTS = List.of(APPLICANT_LIP_HWF_DASHBOARD_NOTIFICATION);
+    private final CoreCaseDataService coreCaseDataService;
```

Comment:
these two fields can be removed as not used in this handler

Reply:
Update from review